tests/org.codechecker.eclipse.rcp.it.tests/src/org/codechecker/eclipse/plugin/utils/GuiUtils.java

```diff
@@ -0,0 +1,205 @@
+package org.codechecker.eclipse.plugin.utils;
```

Comment:
Added a lot of commonly used utility functions for navigation of the user interface.